### PR TITLE
Make PR builder and e2e-gce-parallel e2e runs use the same node count

### LINKED ISSUE #13478: E2e: Services should be able to up and down services
http://kubekins.dls.corp.google.com/view/Critical%20Builds/job/kubernetes-e2e-gce-parallel/7142/

```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/service.go:278
Expected error:
    <*errors.errorString | 0xc20802bf70>: {
        s: "timed out waiting for the condition",
    }
    timed out waiting for the condition
not to have occurred
```

### PR DESCRIPTION
**Pull request by ghost**
Capacity planning this is somewhat hard.  Lets at least be consistent.
Fixes #13478

### CODE REVIEW

**Comment by quinton-hoole** [comment]
@ixdy FYI

**Comment by quinton-hoole** [comment]
Fixes #11675

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -221,6 +222,24 @@ case ${JOB_NAME} in
...
+    : ${GINKGO_TEST_ARGS:="--ginkgo.skip=$(join_regex_allow_empty \
+          ${GCE_PARALLEL_SKIP_TESTS[@]:+${GCE_PARALLEL_SKIP_TESTS[@]}} \
+          ${GCE_PARALLEL_FLAKY_TESTS[@]:+${GCE_PARALLEL_FLAKY_TESTS[@]}} \
+          ${GCE_FLAKY_TESTS[@]:+${GCE_FLAKY_TESTS[@]}} \
+          )"}
+    : ${KUBE_GCE_INSTANCE_PREFIX:="pull-e2e-${EXECUTOR_NUMBER}"}
+    : ${KUBE_GCS_STAGING_PATH_SUFFIX:="-${EXECUTOR_NUMBER}"}
+    : ${PROJECT:="kubernetes-jenkins-pull"}
+    # Override GCE defaults
+    NUM_MINIONS=$NUM_MINIONS_PARALLEL
```
Comment (ihmccreery) [inline hack/jenkins/e2e.sh]: ${NUM_MINIONS_PARALLEL}

File: hack/jenkins/e2e.sh [hunk 2]
```diff
@@ -234,8 +253,8 @@ case ${JOB_NAME} in
           )"}
     : ${KUBE_GCE_INSTANCE_PREFIX:="e2e-test-parallel"}
     : ${PROJECT:="kubernetes-jenkins"}
-    # Override GCE defaults.
-    NUM_MINIONS="6"
+    # Override GCE defaults
+    NUM_MINIONS=$NUM_MINIONS_PARALLEL
```
Comment (ihmccreery) [inline hack/jenkins/e2e.sh]: ${NUM_MINIONS_PARALLEL}

**Comment by ihmccreery** [comment]
LGTM other than style notes.

This is going to be messy to merge with #13531; I'd be okay if you put both of the changes in this PR.

**Comment by quinton-hoole** [comment]
No, lets keep the PR's separate.  I agree with your style nits.  Will do now.

**Comment by quinton-hoole** [comment]
@ihmccreery done, PTAL.

File: hack/jenkins/e2e.sh (hunk 1 above)
Comment (ihmccreery) [inline hack/jenkins/e2e.sh]: This looks like it didn't change.  :cry:

**Comment by ihmccreery** [comment]
Looks like ya missed one.

**Comment by quinton-hoole** [comment]
@ihmccreery yup, sorry.  Done.

**Comment by ihmccreery** [comment]
LGTM; you might want to squash commits.

**Comment by ArtfulCoder** [comment]
@quinton-hoole ping

**Comment by quinton-hoole** [comment]
Yes, my apologies, squashed now.  Abhi, I think that you can safely merge this manually now.  It will hopefully help to unblock your auto-merge queue.

**Comment by ArtfulCoder** [comment]
merged. thanks Q.

**Pull request merged**